### PR TITLE
Fixing formatting and alt description for SEO

### PR DESCRIPTION
**Pull request by amyshwang**

### CODE REVIEW

File: blog/2022-08-05-setting-basic-auth-nginx.md [hunk 1]
```diff
@@ -20,11 +19,12 @@ tags: [tutorial, nginx, security, authentication, proxy]
...
-  height={467}
-  src="/img/blog/2022-08-05/banner.png"
-  width={650}
-></Banner>
+alt="QuestDB log and nginx logo"
+height={467}
+src="/img/blog/2022-08-05/banner.png"
+width={650}
+
+> </Banner>
```
Comment (amyshwang) [inline blog/2022-08-05-setting-basic-auth-nginx.md:27]: Is this actually correct?

File: blog/2022-08-22-using-birch-anomaly-detection-questdb.md [hunk 2]
```diff
@@ -18,11 +17,12 @@ tags: [birch, anomaly detection, python, tutorial]
...
-  width={650}
-></Banner>
+alt="A image of many white sheep and one black sheep, with the QuestDB logo"
+height={360}
+src="/img/blog/2022-08-22/banner.png/"
+width={650}
+
+> </Banner>
 
 In this article, [Fortune Adekogbe](https://www.linkedin.com/in/fortune-adekogbe-a81580176/) introduces how to use the BIRCH algorithm for visual clustering, which assists anomaly detection in QuestDB.
```
Comment (amyshwang) [inline blog/2022-08-22-using-birch-anomaly-detection-questdb.md:27]: Built is complaining about the link here. I wonder if it is to do with the change in L25.

File: blog/2022-08-22-using-birch-anomaly-detection-questdb.md [hunk 3]
```diff
@@ -18,11 +17,12 @@ tags: [birch, anomaly detection, python, tutorial]
...
 
 <Banner
-  alt="detecting the black sheep"
-  height={360}
-  src="/img/blog/2022-08-22/banner.png"
-  width={650}
-></Banner>
+alt="A image of many white sheep and one black sheep, with the QuestDB logo"
+height={360}
+src="/img/blog/2022-08-22/banner.png/"
```
Comment (pswu11) [inline blog/2022-08-22-using-birch-anomaly-detection-questdb.md:22]: no slash here - link to an image, not a hyperlink

File: blog/2022-08-22-using-birch-anomaly-detection-questdb.md [hunk 4]
```diff
@@ -34,37 +34,37 @@ BIRCH is also used in scikit-learn, the popular Python package that makes machin
...
 
 ## Implementing BIRCH and QuestDB
 
-To better understand BIRCH and how it can be used with QuestDB, you will solve a sample anomaly detection problem. 
+To better understand BIRCH and how it can be used with QuestDB, you will solve a sample anomaly detection problem.
 
-### Setting up QuestDB 
+### Setting up QuestDB
 
 You have several options for installing QuestDB, including Docker and Homebrew. [Check the documentation](/docs) for details.
```
Comment (pswu11) [inline blog/2022-08-22-using-birch-anomaly-detection-questdb.md:69]: /docs/

File: blog/2022-08-05-setting-basic-auth-nginx.md [hunk 5]
```diff
@@ -189,24 +186,24 @@ After reloading the configuration file, open a new tab in your browser to test i
 
 `localhost` points to QuestDB based on the `nginx.conf`.
 
-![](/img/blog/2022-08-05/step_4_1.png)
+![](/img/blog/2022-08-05/step_4_1.png/)
```
Comment (amyshwang) [inline blog/2022-08-05-setting-basic-auth-nginx.md:189]:
```suggestion
![](/img/blog/2022-08-05/step_4_1.png)
```

File: blog/2022-08-05-setting-basic-auth-nginx.md [hunk 6]
```diff
@@ -189,24 +186,24 @@ After reloading the configuration file, open a new tab in your browser to test i
...
 
 If `localhost` binding is working correctly, you should see the following screen asking for your username and password to log into QuestDB:
 
-![](/img/blog/2022-08-05/step_4_2.png)
+![](/img/blog/2022-08-05/step_4_2.png/)
```
Comment (amyshwang) [inline blog/2022-08-05-setting-basic-auth-nginx.md:193]:
```suggestion
![](/img/blog/2022-08-05/step_4_2.png)
```

File: blog/2022-08-05-setting-basic-auth-nginx.md [hunk 7]
```diff
@@ -189,24 +186,24 @@ After reloading the configuration file, open a new tab in your browser to test i
...
 
 Once you enter your username and password, Nginx will go back to `.htpasswd` to verify whether you’ve entered the correct credentials. If the authentication succeeds, you’ll be able to log onto QuestDB and carry on with your work, as shown below:
 
-![](/img/blog/2022-08-05/step_4_3.png)
+![](/img/blog/2022-08-05/step_4_3.png/)
```
[the remark this answers is not in the document]
Comment (amyshwang) [reply blog/2022-08-05-setting-basic-auth-nginx.md:197]:
```suggestion
![](/img/blog/2022-08-05/step_4_3.png)
```

File: blog/2022-08-05-setting-basic-auth-nginx.md [hunk 8]
```diff
@@ -189,24 +186,24 @@ After reloading the configuration file, open a new tab in your browser to test i
...
 
 However, if the authentication fails, you’ll see the following screen with an HTTP 401 specifying that you need the authorization to access `localhost`.
 
-![](/img/blog/2022-08-05/step_4_4.png)
+![](/img/blog/2022-08-05/step_4_4.png/)
```
Comment (amyshwang) [inline blog/2022-08-05-setting-basic-auth-nginx.md:201]:
```suggestion
![](/img/blog/2022-08-05/step_4_4.png)
```

File: blog/2022-08-05-setting-basic-auth-nginx.md [hunk 9]
```diff
@@ -38,26 +37,24 @@ Data privacy and security is one of the most critical areas of concern when work
...
 
-There are several ways to achieve this, such as [SSH tunneling](https://en.wikipedia.org/wiki/Tunneling_protocol), [OAuth](https://oauth.net/), [token-based auth](https://www.okta.com/au/identity-101/what-is-token-based-authentication/), etc. However, this article will take you through the most straightforward authentication setup of them so that you can get started with basic authentication with minimal effort using Nginx.
-
+There are several ways to achieve this, such as [SSH tunneling](https://en.wikipedia.org/wiki/Tunneling_protocol/), [OAuth](https://oauth.net/), [token-based auth](https://www.okta.com/au/identity-101/what-is-token-based-authentication/), etc. However, this article will take you through the most straightforward authentication setup of them so that you can get started with basic authentication with minimal effort using Nginx.
```
Comment (pswu11) [inline blog/2022-08-05-setting-basic-auth-nginx.md:40]: remove `/` as external links won't matter - and perhaps could lead to something broken if they don't have redirect.

File: blog/2022-08-05-setting-basic-auth-nginx.md [hunk 10]
```diff
@@ -38,26 +37,24 @@ Data privacy and security is one of the most critical areas of concern when work
...
-While you can set up authentication for QuestDB open source, [QuestDB Cloud](/cloud) offers built-in authentication that offers a hassle-free, Out-of-the-Box solution.
+While you can set up authentication for QuestDB open source, [QuestDB Cloud](/cloud/) offers built-in authentication that offers a hassle-free, Out-of-the-Box solution.
 
 :::
 
-
 ## How Basic Authentication Works in Nginx
 
-Nginx is a multi-purpose application that can be used as a [reverse proxy](https://microsoft.github.io/reverse-proxy/articles/authn-authz.html), a load balancer, or an HTTP cache. This means if you direct all your incoming traffic for QuestDB to Nginx, it can decide what action to take on that request.
+Nginx is a multi-purpose application that can be used as a [reverse proxy](https://microsoft.github.io/reverse-proxy/articles/authn-authz.html/), a load balancer, or an HTTP cache. This means if you direct all your incoming traffic for QuestDB to Nginx, it can decide what action to take on that request.
```
Comment (pswu11) [inline blog/2022-08-05-setting-basic-auth-nginx.md:50]: link broken remove `/ `

**Comment by amyshwang** [comment]
Close as we're looking into overall SEO strategies.

**Pull request closed without merging**